src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,49 @@
+## Distributed Tracing
+
+Traces can bridge across multiple software services. Each span in a trace can be represented as a `sentry-trace` header, containing the trace id, span id, and sampling details. This `sentry-trace` header can be passed to downstream services so that they can create spans that are a continuation of the trace started in the originating service.
+
+To obtain a trace header from the span, use `ISpan.GetTraceHeader()` method, then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
+
+```csharp
+using System;
+using System.Net.Http;
+using Sentry;
+
+var transaction = SentrySdk.StartTransaction("name", "op");
+var sentryTraceHeader = transaction.GetTraceHeader();
+
+using var request = new HttpRequestMessage(HttpMethod.Get, "https://example.com");
+request.Headers.Add("sentry-trace", sentryTraceHeader.ToString());
+```
+
+<Note>
+
+Sentry also provides `SentryHttpMessageHandler`, which can be used to automatically attach trace headers to outgoing HTTP requests. Learn more about it in <PlatformLink to="/performance/included-instrumentation/">this article</PlatformLink>.
+
+</Note>
+
+To create a span as a continuation of the trace retrieved from the upstream service, pass the `sentry-trace` header value to the `StartTransaction(...)` method:
+
+```csharp
+using System;
+using Sentry;
+
+var httpContext = ...;
+var sentryTraceHeaderValue = httpContext.Request.Headers["sentry-trace"].ToString();
```

Comment:
if a request coming in without Sentry's header, this would throw.
I rather we avoid such code snippet. Can we do something else like?:

```suggestion
if (!httpContext.Request.Headers.TryGetValue("sentry-trace", out var sentryTraceHeaderValue))
{
    return; // No Sentry header in this request
}
```
To be clear users can't assume such header will always be there?

Reply:
Done. Inverted it to avoid `return`, I think it's clearer that way.